pandas/tests/io/formats/test_to_html.py

```diff
@@ -1905,6 +2013,20 @@ def test_to_html_multiindex_max_cols(self):
         </table>""")
         assert result == expected
 
+    @pytest.mark.parametrize('header', [True])
```

Comment:
If these are always just True what's the point of parametrization?

Reply:
The tests added here are the same tests added for regression testing of #22655. If either the `index`, `header` or `index_names` parameter is `False`, the tests will currently fail.

So the tests for regression testing of the refactoring only covers the `index=True`, `header=True`, `index_names=True` codepath.

I appreciate that the changes to the tests are not ready for merging into master but the tests can be removed from this PR before approving the refactoring.

Reply:
I see. If that's the case just go ahead and remove here and keep as part of #22655. Relatively minor but would keep a cleaner and more logical history of changes that way